ext/hiredis_ext/connection.c

```diff
@@ -2,6 +2,14 @@
 #include <errno.h>
 #include "hiredis_ext.h"
 
+struct redisSSLContext {
```

Comment:
This is a bit of an ugly hack since it's reaching inside the hiredis internals to get this information.

I think we're better off modifying hiredis to support the setting of verify mode: https://github.com/redis/hiredis/pull/1085